generators/app/templates/package.json

```diff
@@ -31,7 +31,9 @@
   "dependencies": {
     "classnames": "^2.2.5",
     "react": "^15.4.2",
-    "terra-mixins": "^1.0.0"
+    "terra-mixins": "^1.0.0",
+    "terra-props-table": "0.1.0",
+    "terra-markdown": "0.1.0",
```

Comment:
Although I'm not sure why travis CI didn't have a problem with this, running locally this trailing comma in the package.json fails on npm install